source/comply/audit-log.rst

```diff
@@ -3,39 +3,205 @@ Audit Log v2 (Experimental) (E20)
+Configuring the Mattermost server to use the new audit log requires editing the ``config.json`` file directly. The audit settings output audit records to allow any combination of local file, syslog, and TCP socket targets.
+
+File target supports rotation and compression triggered by size and/or duration. Syslog target supports local and remote syslog servers, with or without TLS transport. TCP socket target can be configured with an IP address or domain name, port, and optional TLS certificate.
 
+.. note::
+   Mattermost v6.0 introduced a change to the logging engine library which changes how the logger is configured in Mattermost 5.x versions. Please refer to :download:`Advanced Logging sample file <../samples/sample-logger-config.json>` for a sample configuration file supported in in v6.0 and beyond. 
+
+   
 **Accessing configuration options for audit log**
 
-Open ``config.json`` and navigate to the audit settings. The following `configuration options <https://docs.mattermost.com/configure/configuration-settings.html#audit-settings>`_ are available:
+Open ``config.json`` and navigate to the audit settings under ``ExperimentalAuditSettings``. Within the setting ``AdvancedLoggingConfig`` you are able to specifiy a filespec to another config file, a database DSN, or JSON. 
+
+The logging configuration JSON is an object (unordered collection) containing names and log target values. Each log target contains a type, options specific to the target type, format, and levels.
+
+The example below specifies one log target that outputs to the console using a plain text format with pipes delimiting fields. The standard log levels are listed, with only `error` and lower outputing a stack trace. The `error` level will output in red for log targets and formatters that support color output.
+
+.. code-block:: json
+
+  {
+      "sample-console": {
+        "type": "console",
+        "options": {
+          "out": "stdout"
+        },
+        "format": "plain",
+        "format_options": {
+          "delim": " | "
+        },
+        "levels": [
+          {"id": 5, "name": "debug"},
+          {"id": 4, "name": "info"},
+          {"id": 3, "name": "warn"},
+          {"id": 2, "name": "error", "stacktrace": true, "color": 31},
+          {"id": 1, "name": "fatal", "stacktrace": true},
+          {"id": 0, "name": "panic", "stacktrace": true}
+        ],
+        "maxqueuesize": 1000
+      }
+    }
+    
+.. Note::
+    Filenames for ``AdvancedLoggingConfig`` can contain an absolute filename, a relative filename, or embedded JSON.
+
+See the :download:`Advanced Logging Options Sample JSON ZIP file <../samples/advanced-logging-options-sample-json.zip>` for a sample configuration file.  
+
+Log Target Types
+~~~~~~~~~~~~~~~~
+Log target types include Console, Syslog, File, or TCP. 
+
+**Console configuration options:**
```

Comment:
These sections may be good candidates for using a tabbed content approach, though I'm not 100% sure that CSV-based tables will display as intended. The basic format goes like this:

```
.. tabs::
   
    .. tab:: tabname

    .. tab:: tabname

    .. tab:: tabname
```

I'm happy to help format this content as tabs and see if CSV-based tables display as intended. Let me know if you're open to this approach.

Reply:
Let's do the tabbed content post 6.0 when we re-vamp the configuration sections.  I am sure this one will get re-reviewed at that time.